frontend/src/components/utils/NavBar/index.tsx

```diff
@@ -69,7 +68,7 @@ const useStyles = makeStyles(() => ({
     padding: '20px 30px',
   },
   homeImage: {
-    marginBottom: '8px',
+    marginBottom: '-8px',
```

Comment:
Is there a way to do this without using negative margins, and instead adding a margin bottom to the element on top?